rql/README.md

```diff
@@ -0,0 +1,128 @@
+# RQL (Rest Query Language)
+
+A library to parse support advanced REST API query parameters like (filter, pagination, sort, group, search) and logical operators on the keys (like eq, neq, like, gt, lt etc)
+
+It takes a Golang struct and a json string as input and returns a Golang object that can be used to prepare SQL Statements (using raw sql or ORM Query builders).
+
+### Usage
+
+Frontend should send the parameters and operator like this schema to the backend service on some route with `POST` HTTP Method
+
+```json
+{
+  "filters": [
+    { "name": "id", "operator": "neq", "value": 20 },
```

Comment:
if we are taking type can we include the type field in the example as well

Reply:
the payload is not supposed to have a type field as input. It's derived from the struct tags.

Reply:
It can be sent but not used.